### PR TITLE
updated analysis json files

### PR DESCRIPTION
**Pull request by HelTob**
Updated section about "provided test results" in the DE and EN json files.

### CODE REVIEW

File: src/data/analyse_en.json [hunk 1]
```diff
@@ -78,10 +78,10 @@
             "totalKey": ["app_downloads_cumulated"]
         },
         {
-            "title": "Transmitted and registered test results",
+            "title": "Provided test results",
```
Comment (MikeMcC399) [inline src/data/analyse_en.json:81]: I suggest changing the word order

"Test results provided"

which sounds more English.

File: src/data/analyse_en.json [hunk 2]
```diff
@@ -78,10 +78,10 @@
...
+                "content": "<strong>Provided test results</strong><p>Count of PCR test results and antigen rapid test results provided by laboratories and rapid test sites on the Corona-Warn-App server, as well as the sum of test results of both test types.</p><strong>Provided test results: PCR tests</strong><p>Count of positive, negative and invalid as well as the sum of all PCR test results provided by laboratories and rapid test sites on the Corona-Warn-App server.</p><strong>Provided test results: Rapid antigen tests</strong><p>Count of positive, negative and invalid as well as the sum of all rapid antigen test results provided by laboratories and rapid test sites on the Corona-Warn-App server.</p><strong>Time intervals</strong><p>Daily: Count accurate to the day<br />Weekly: Weekly count by calendar week (CW)<br />7-day moving average: 7-day moving average accurate to the day<br />Cumulative: Cumulative sum accurate to the day, since the beginning of the measurement</p><strong> Reference date</strong><p>The reference date is the day of the test submission to the test server.</p><strong> Data sources </strong><p>Corona-Warn-App backend</p>"
```
Comment (MikeMcC399) [inline src/data/analyse_en.json:84]: See above also if you decide to change the word order in the title, then also in the title of the text here i.e. "Test results provided"

**Review by MikeMcC399 — COMMENTED** [review]
I made a style suggestion for the title: "Provided test results" to change the word order.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for the review, we will adopt the text.

**Review by dsarkar — APPROVED** [review]

**Comment by HelTob** [comment]
Thank you for your suggestions for improvement, which I appreciate very much.

**Comment by MikeMcC399** [comment]
@HelTob 
> Thank you for your suggestions for improvement, which I appreciate very much.

I'm happy to assist! 🙂

**Comment by MikeMcC399** [comment]
@dsarkar / @HelTob 

The label on the graph itself did not change. Was it intended **not** to change this label and only to change the information text?

image

**Comment by HelTob** [comment]
When refreshing the page, it briefly shows the correct title. Reminds me of #2171

**Comment by MikeMcC399** [comment]
@HelTob 

I imagine that you would need to change the text also in two other places:

https://github.com/corona-warn-app/cwa-website/blob/9446409f1adb06ef4d550131bf5c9be16b69fefc/src/data/analyse_en.json#L140

https://github.com/corona-warn-app/cwa-website/blob/9446409f1adb06ef4d550131bf5c9be16b69fefc/src/data/analyse_de.json#L140

if you want to change the labels on the graphs themselves.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for pointing the issue with the titles and proving the solution!

**Pull request merged**